openshift/e2e-tests-openshift.sh

```diff
@@ -196,7 +197,21 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
```

Comment:
Kept the file in repo, the following method will change as we will have 1.2.0 operator release.